Greek mask effort

```diff
@@ -0,0 +1,4 @@
+Team based in Thessaloniki, Greece is producing masks for medical use.
+Translated page here: https://translate.google.com/translate?hl=&sl=el&tl=en&u=https%3A%2F%2Fcovid19response.gr%2Fproject1.html
```

Comment:
I get: `Server Error 404 Page Not Found`on the link

Reply:
@bneiluj
bneiluj 9 minutes ago Owner
I get: Server Error 404 Page Not Foundon the link

@MorganGR	Reply…

Original Page here: https://covid19response.gr/project1.html

Reply:
It is based on the Prusa version (and improved)